src/index.css

```diff
@@ -1,27 +1,73 @@
-body {
+:root {
+  --screen-color: #858693;
+  --dark: #000;
+  --white: #fff;
+  --operator-color: #eb8537;
+}
+
+* {
+  box-sizing: border-box;
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  padding: 0;
+  border-radius: 0;
+}
+
+#root {
+  text-align: center;
+}
+
+.container {
+  background-color: #fff;
+  height: auto;
+  margin: 40px;
+  max-width: 480px;
+  padding: 10px;
+  width: 100%;
+}
+
+.rowBtn {
+  width: 400px;
+  display: grid;
+}
+
+.normal {
+  grid-template-columns: repeat(4, 1fr);
+}
+
+.double {
+  grid-template-columns: 2fr 1fr 1fr;
+}
+
+.screen-row input {
+  max-width: 480px;
+  background-color: var(--screen-color);
+  border: 0;
+  color: rgb(146, 143, 143);
+  padding: 10px;
+  text-align: right;
+  display: flex;
+  justify-content: flex-end;
+  font-size: 2.28rem;
+  height: 76px;
+}
```

Comment:
- [ ] This input field is not properly aligned. This is caused by the width properties given to it and also the one is given to `.rowBtn`. We will talk about what to do with `.rowBtn` next.

  ![Screenshot 2022-06-29 021925](https://user-images.githubusercontent.com/45002654/176321618-18a32699-24a6-4073-9755-827e8be86f28.png)


  Let's fist remove the `max-width` and instead give it a `width: 100%`.